tests/src/test/scala/cats/tests/AlternativeSuite.scala

```diff
@@ -11,4 +19,10 @@ class MonadCombineSuite extends CatsSuite {
       Alternative[List].separate(list) should === (expected)
     }
   }
+
+  test("guard") {
+    forAll { (b: Boolean) =>
+      Alternative[Option].guard(b).isDefined should === (b)
```

Comment:
This test case will probably run 100 times, but only two values are actually tested. I'd rather generate different valid `Alternative` instances.

Reply:
I'm not sure how to do that with `forall`. Do you have an example testcase I can look into and copy the behavior of?

Reply:
You don’t need forall. Just write a def for the law, then call it for two inputs.